Traceback (most recent call last):
  File "C:\Users\user\miniconda3\envs\ds\Lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\user\miniconda3\envs\ds\Lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\miniconda3\envs\ds\Lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\miniconda3\envs\ds\Lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\miniconda3\envs\ds\Lib\asyncio\base_events.py", line 650, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\user\miniconda3\envs\ds\Lib\contextlib.py", line 222, in __aexit__
    await self.gen.athrow(typ, value, traceback)
  File "C:\Users\user\miniconda3\envs\ds\Lib\site-packages\nbclient\client.py", line 608, in async_setup_kernel
    yield
  File "C:\Users\user\miniconda3\envs\ds\Lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\user\miniconda3\envs\ds\Lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\user\miniconda3\envs\ds\Lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Convertir a tipo datetime
df_pollutants[df_pollutants.columns[0]] = pd.to_datetime(df_pollutants.timestamp)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mValueError[0m                                Traceback (most recent call last)
Cell [1;32mIn [37], line 2[0m
[0;32m      1[0m [38;5;66;03m# Convertir a tipo datetime[39;00m
[1;32m----> 2[0m df_pollutants[df_pollutants[38;5;241m.[39mcolumns[[38;5;241m0[39m]] [38;5;241m=[39m [43mpd[49m[38;5;241;43m.[39;49m[43mto_datetime[49m[43m([49m[43mdf_pollutants[49m[38;5;241;43m.[39;49m[43mtimestamp[49m[43m)[49m

File [1;32m~\miniconda3\envs\ds\Lib\site-packages\pandas\core\tools\datetimes.py:1064[0m, in [0;36mto_datetime[1;34m(arg, errors, dayfirst, yearfirst, utc, format, exact, unit, infer_datetime_format, origin, cache)[0m
[0;32m   1062[0m             result [38;5;241m=[39m arg[38;5;241m.[39mtz_localize(tz)
[0;32m   1063[0m [38;5;28;01melif[39;00m [38;5;28misinstance[39m(arg, ABCSeries):
[1;32m-> 1064[0m     cache_array [38;5;241m=[39m [43m_maybe_cache[49m[43m([49m[43marg[49m[43m,[49m[43m [49m[38;5;28;43mformat[39;49m[43m,[49m[43m [49m[43mcache[49m[43m,[49m[43m [49m[43mconvert_listlike[49m[43m)[49m
[0;32m   1065[0m     [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m cache_array[38;5;241m.[39mempty:
[0;32m   1066[0m         result [38;5;241m=[39m arg[38;5;241m.[39mmap(cache_array)

File [1;32m~\miniconda3\envs\ds\Lib\site-packages\pandas\core\tools\datetimes.py:229[0m, in [0;36m_maybe_cache[1;34m(arg, format, cache, convert_listlike)[0m
[0;32m    227[0m unique_dates [38;5;241m=[39m unique(arg)
[0;32m    228[0m [38;5;28;01mif[39;00m [38;5;28mlen[39m(unique_dates) [38;5;241m<[39m [38;5;28mlen[39m(arg):
[1;32m--> 229[0m     cache_dates [38;5;241m=[39m [43mconvert_listlike[49m[43m([49m[43munique_dates[49m[43m,[49m[43m [49m[38;5;28;43mformat[39;49m[43m)[49m
[0;32m    230[0m     [38;5;66;03m# GH#45319[39;00m
[0;32m    231[0m     [38;5;28;01mtry[39;00m:

File [1;32m~\miniconda3\envs\ds\Lib\site-packages\pandas\core\tools\datetimes.py:438[0m, in [0;36m_convert_listlike_datetimes[1;34m(arg, format, name, tz, unit, errors, infer_datetime_format, dayfirst, yearfirst, exact)[0m
[0;32m    436[0m [38;5;28;01massert[39;00m [38;5;28mformat[39m [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m [38;5;129;01mor[39;00m infer_datetime_format
[0;32m    437[0m utc [38;5;241m=[39m tz [38;5;241m==[39m [38;5;124m"[39m[38;5;124mutc[39m[38;5;124m"[39m
[1;32m--> 438[0m result, tz_parsed [38;5;241m=[39m [43mobjects_to_datetime64ns[49m[43m([49m
[0;32m    439[0m [43m    [49m[43marg[49m[43m,[49m
[0;32m    440[0m [43m    [49m[43mdayfirst[49m[38;5;241;43m=[39;49m[43mdayfirst[49m[43m,[49m
[0;32m    441[0m [43m    [49m[43myearfirst[49m[38;5;241;43m=[39;49m[43myearfirst[49m[43m,[49m
[0;32m    442[0m [43m    [49m[43mutc[49m[38;5;241;43m=[39;49m[43mutc[49m[43m,[49m
[0;32m    443[0m [43m    [49m[43merrors[49m[38;5;241;43m=[39;49m[43merrors[49m[43m,[49m
[0;32m    444[0m [43m    [49m[43mrequire_iso8601[49m[38;5;241;43m=[39;49m[43mrequire_iso8601[49m[43m,[49m
[0;32m    445[0m [43m    [49m[43mallow_object[49m[38;5;241;43m=[39;49m[38;5;28;43;01mTrue[39;49;00m[43m,[49m
[0;32m    446[0m [43m[49m[43m)[49m
[0;32m    448[0m [38;5;28;01mif[39;00m tz_parsed [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:
[0;32m    449[0m     [38;5;66;03m# We can take a shortcut since the datetime64 numpy array[39;00m
[0;32m    450[0m     [38;5;66;03m# is in UTC[39;00m
[0;32m    451[0m     dta [38;5;241m=[39m DatetimeArray(result, dtype[38;5;241m=[39mtz_to_dtype(tz_parsed))

File [1;32m~\miniconda3\envs\ds\Lib\site-packages\pandas\core\arrays\datetimes.py:2177[0m, in [0;36mobjects_to_datetime64ns[1;34m(data, dayfirst, yearfirst, utc, errors, require_iso8601, allow_object, allow_mixed)[0m
[0;32m   2175[0m order: Literal[[38;5;124m"[39m[38;5;124mF[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124mC[39m[38;5;124m"[39m] [38;5;241m=[39m [38;5;124m"[39m[38;5;124mF[39m[38;5;124m"[39m [38;5;28;01mif[39;00m flags[38;5;241m.[39mf_contiguous [38;5;28;01melse[39;00m [38;5;124m"[39m[38;5;124mC[39m[38;5;124m"[39m
[0;32m   2176[0m [38;5;28;01mtry[39;00m:
[1;32m-> 2177[0m     result, tz_parsed [38;5;241m=[39m [43mtslib[49m[38;5;241;43m.[39;49m[43marray_to_datetime[49m[43m([49m
[0;32m   2178[0m [43m        [49m[43mdata[49m[38;5;241;43m.[39;49m[43mravel[49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43mK[39;49m[38;5;124;43m"[39;49m[43m)[49m[43m,[49m
[0;32m   2179[0m [43m        [49m[43merrors[49m[38;5;241;43m=[39;49m[43merrors[49m[43m,[49m
[0;32m   2180[0m [43m        [49m[43mutc[49m[38;5;241;43m=[39;49m[43mutc[49m[43m,[49m
[0;32m   2181[0m [43m        [49m[43mdayfirst[49m[38;5;241;43m=[39;49m[43mdayfirst[49m[43m,[49m
[0;32m   2182[0m [43m        [49m[43myearfirst[49m[38;5;241;43m=[39;49m[43myearfirst[49m[43m,[49m
[0;32m   2183[0m [43m        [49m[43mrequire_iso8601[49m[38;5;241;43m=[39;49m[43mrequire_iso8601[49m[43m,[49m
[0;32m   2184[0m [43m        [49m[43mallow_mixed[49m[38;5;241;43m=[39;49m[43mallow_mixed[49m[43m,[49m
[0;32m   2185[0m [43m    [49m[43m)[49m
[0;32m   2186[0m     result [38;5;241m=[39m result[38;5;241m.[39mreshape(data[38;5;241m.[39mshape, order[38;5;241m=[39morder)
[0;32m   2187[0m [38;5;28;01mexcept[39;00m [38;5;167;01mOverflowError[39;00m [38;5;28;01mas[39;00m err:
[0;32m   2188[0m     [38;5;66;03m# Exception is raised when a part of date is greater than 32 bit signed int[39;00m

File [1;32m~\miniconda3\envs\ds\Lib\site-packages\pandas\_libs\tslib.pyx:427[0m, in [0;36mpandas._libs.tslib.array_to_datetime[1;34m()[0m

File [1;32m~\miniconda3\envs\ds\Lib\site-packages\pandas\_libs\tslib.pyx:698[0m, in [0;36mpandas._libs.tslib.array_to_datetime[1;34m()[0m

[1;31mValueError[0m: mixed datetimes and integers in passed array
ValueError: mixed datetimes and integers in passed array

